spec/features/admin/enterprises_spec.rb

```diff
@@ -286,6 +286,27 @@
     end
   end
 
+  context "as an Enterprise manager", js: true do
+    let(:enterprise_manager) { create(:user) }
+    let(:enterprise) { create(:enterprise, users: [:enterprise_manager]) }
+
+    before do
+      login_as enterprise_manager
+      visit admin_enterprises_path
+
+      it "allows me to edit notifications" do
+        expect(page).to have_content enterprise.name
+
+        accept_alert do
+          within(".side_menu") { click_link "Users" }
+        end
+
+        select2_select enterprise_manager.email, from: 'receives_notifications_dropdown'
```

Comment:
This is intended to test that there is a dropdown for the email address to receive notifications, but I'm not sure it's actually working as intended.